### PR TITLE
functionality of result page

### PR DESCRIPTION
**Pull request by hibaakel**
#97

### CODE REVIEW

File: helpers/convertDis.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const convertoption = (item) => {
+  item = item.toLowerCase()
+  var options = []
+  if (item === 'visual') {
+    options = ['Audio Recordings', 'Braille', 'Big Fonts', 'Carer', 'Place for Guide Dog', 'SMS messaging']
+  }
+  if (item === 'aural') {
```
Comment (m4v15) [inline helpers/convertDis.js]: Is there a reason that you use if and not if/else statements, and then return in each section?

You could do:
```
  if (item === 'mental') {
    return  ['Good Lighting', 'Carer']
  }
  else (item === 'physical') {
    return ['WheelChair Access', 'Disabled Parking', 'Disabled Toilets', 'Carer']
  }
```

Comment (m4v15) [inline helpers/convertDis.js]: I just wrote that quickly, check it works if you implement it!

File: text.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ module.exports = {
     homeIcon: 'Icon for the home page',
     helpIcon: 'Icon for the help button',
     helpInfo: 'easyNaz is made in aim to let people know of the few accissible places in Nazareth and encourage more businesses to become accessible! In our app you can find all the accessible places and businesses in Nazrareth anytime and just a few clicks depends on what you need or looking for.',
+    notFound: 'Sorry we have no services in our databse that match your selections',
```
Comment (m4v15) [inline text.js]: "database" not "databse"

Comment (MohamedOmarii) [inline text.js]: I fixed :+1:  @m4v15

File: app.js [hunk 3]
```diff
@@ -28,6 +32,9 @@ app.set('views', path.join(__dirname, './', 'views'))
 
 app.use(router)
 
-app.listen(app.get('port'), () => {
-  console.log('Server running on port:', app.get('port'))
+db.on('error', console.error.bind(console, 'connection error:'))
```
Comment (Suuuuha) [inline app.js]: why did you use console.error.bind... in  here?

Comment (MohamedOmarii) [inline app.js]: ask @m4v15

Comment (m4v15) [inline app.js]: because thats what the mongoose docs used

Comment (m4v15) [inline app.js]: Its basically an asynchronous way of handling an error by printing to the console

File: routes/results.js [hunk 4]
```diff
@@ -1,9 +1,24 @@
-const fs = require('fs')
+const dbFunctions = require('../database/db_functions.js')
+const convertFunctions = require('../helpers/convertDis.js')
+module.exports = (req, res) => {
+  const disabilityOption = req.query.disability
+  const serviceOption = convertFunctions.capitalise(req.query.service)
```
Comment (m4v15) [inline routes/results.js]: due to an error on our part we have set up the query string to use "services" not "service" so I will change this here to make sure it works

**Pull request merged**